src/App.jsx

```diff
@@ -1,80 +1,16 @@
 import React from 'react';
 import './App.css';
+import Article from './components/Article/Article';
+import Header from './components/Header/Header';
+import Welcome from './components/Welcome/Welcome';
 // Move each BEM block to a separate component (file) and import them here
```

Comment:
You can optimize your imports using `index.js` for each component

```suggestion

import Article from './components/Article';
import Header from './components/Header';
import Welcome from './components/Welcome';

```